tasks/main.yml

```diff
@@ -28,3 +28,11 @@
     - upsd.conf
     - upsmon.conf
   notify: restart nut
+
+- name: Install custom notifycmd script.
+  copy:
+    dest: "{{ nut_upsmon_notifycmd }}"
+    content: "{{ nut_upsmon_notifycmd_content }}"
+    owner: root
+    group: nut
+    mode: 0770
```

Comment:
To avoid emptying a script when not using the content, I would like a check for activating this task only when `nut_upsmon_notifycmd_content` is defined, something like:
```
    when: {{ nut_upsmon_notifycmd_content | length }} > 0
```

Reply:
@ntd that was actually the idea but i forgot to implement this, since i was not using the functionality.

I just added a check for `nut_upsmon_notifycmd_content` to be defined.